frontend/src/static/js/pages/styles/MediaPage.scss

```
@@ -1245,9 +1245,9 @@ $widerLayoutVersion: true; // @todo: Choose dynamically the layout version, base
 								text-align: initial;
 
 								box-shadow:
-									rgba(#000, 0.14) 0px 16px 24px 2px,
-									rgba(#000, 0.12) 0px 6px 30px 5px,
-									rgba(#000, 0.4) 0px 8px 10px;
+									rgb(0 12 32 / 0.14) 0px 16px 24px 2px,
+									rgb(0 12 32 / 0.12) 0px 6px 30px 5px,
+									rgb(0 12 32 / 0.4) 0px 8px 10px;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)